config/config.go

```diff
@@ -432,6 +435,8 @@ func (cfg Config) Validate() error {
 		validateKey("Twilio.AccountSID", cfg.Twilio.AccountSID),
 		validateKey("Twilio.AuthToken", cfg.Twilio.AuthToken),
 		validateKey("Twilio.AlternateAuthToken", cfg.Twilio.AlternateAuthToken),
+		validate.ASCII("Twilio.VoiceName", cfg.Twilio.VoiceName, 0, 50),
+		validate.ASCII("Twilio.VoiceLanguage", cfg.Twilio.VoiceLanguage, 0, 10),
```

Comment:
placeholder text on these could be useful. I wasn't totally sure if `english` would work, then I saw a test later on during the review using `en-US`

Reply:
@Forfold I didn't see any other examples of default values being used, so by doing so would deviate from the current config patterns.  If these values aren't set (like how it is currently today), it'll use the default setting within Twilio itself, so these don't need to be set at all.